pkgs/top-level/all-packages.nix

```
@@ -6023,6 +6023,8 @@ with pkgs;
 
   starboard = callPackage ../applications/networking/cluster/starboard { };
 
+  stats = callPackage ../tools/system/stats { };
```

```suggestion
  stats = callPackage ../os-specific/darwin/stats { };
```
It may be more appropriate under "os-specific/darwin/stats".